src/applications/personalization/dashboard/components/apply-for-benefits/ApplicationInProgress.jsx

```diff
@@ -24,48 +25,68 @@ const ApplicationInProgress = ({
   lastOpenedDate,
   presentableFormId,
 }) => {
-  return (
-    <DashboardWidgetWrapper>
-      <div
-        className="vads-u-display--flex vads-u-margin-bottom--3"
-        data-testid="application-in-progress"
-      >
-        <div className="vads-u-display--flex vads-u-width--full vads-u-flex-direction--column vads-u-justify-content--space-between vads-u-align-items--flex-start vads-u-background-color--gray-lightest vads-u-padding--2p5">
-          <div className="vads-u-width--full">
-            <p
-              id={formId}
-              className="vads-u-text-transform--uppercase vads-u-margin-y--0"
-            >
-              {presentableFormId}
+  const content = (
+    <>
+      <div className="vads-u-width--full">
+        <p
+          id={formId}
+          className="vads-u-text-transform--uppercase vads-u-margin-y--0"
+        >
+          {presentableFormId}
+        </p>
+        <h3
+          aria-describedby={formId}
+          className="vads-u-font-size--h3 vads-u-margin-top--0"
+        >
+          {capitalizeFirstLetter(formTitle)}
```

Comment:
Side note: this utility function could get moved to a common location instead of being at the beginning of this file. Orrrrrr you could use[ lodash's capitalize function](https://lodash.com/docs/4.17.15#capitalize) instead and get rid of this home spun util altogether.